### PR TITLE
[feature] gtfs : read as collections

### PR DESCRIPTION
**Pull request by patochectp**

### CODE REVIEW

File: src/gtfs/mod.rs [hunk 1]
```diff
@@ -413,20 +424,58 @@ impl Reader {
             ))
         }
     }
+    /// Imports `Collections` from the
+    /// [GTFS](https://gtfs.org/reference/static).
+    /// files in the given directory.
+    /// This method will try to detect if the input is a zipped archive or not.
+    /// If the default file type mechanism is not enough, you can use
+    /// [Reader::parse_zip] or [Reader::parse_dir].
```
Comment (ArnaudOggy) [inline src/gtfs/mod.rs:432]:
```suggestion
    /// [Reader::parse_zip_collections] or [Reader::parse_dir_collections].
```

**Review by ArnaudOggy — DISMISSED** [review]

File: src/gtfs/mod.rs [hunk 2]
```diff
@@ -413,20 +424,58 @@ impl Reader {
...
     /// Imports a `Model` from the [GTFS](https://gtfs.org/reference/static)
     /// files in the `path` directory.
     pub fn parse_dir(self, path: impl AsRef<Path>) -> Result<Model> {
+        let collections = self.parse_dir_collections(path)?;
+        Model::new(collections)
+    }
+
+    /// Imports `Collections` from the [GTFS](https://gtfs.org/reference/static)
+    /// files in the `path` directory.
+    fn parse_dir_collections(self, path: impl AsRef<Path>) -> Result<Collections> {
```
Comment (ArnaudOggy) [inline src/gtfs/mod.rs:468]: We can push the idea further and put them in `pub`

Comment (patochectp) [reply src/gtfs/mod.rs:468]: This is not necessary because `parse_collections` is `pub`

Comment (ArnaudOggy) [reply src/gtfs/mod.rs:468]: Yes I saw. It's in case we needed it later. and to remain homogeneous.
But yes for the moment we can stay like that.

**Review by ArnaudOggy — APPROVED** [review]

**Review by ArnaudOggy — APPROVED** [review]

**Pull request merged**